WalletWasabi.Fluent/Behaviors/ClipboardCopyBehavior.cs

```diff
@@ -0,0 +1,83 @@
+using System.Reactive;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using System.Threading.Tasks;
+using Avalonia;
+using Avalonia.Controls;
+using Avalonia.Input;
+using Avalonia.Interactivity;
+using ReactiveUI;
+using WalletWasabi.Fluent.Extensions;
+
+namespace WalletWasabi.Fluent.Behaviors;
+
+public class ClipboardCopyBehavior : AttachedToVisualTreeBehavior<Control>
```

Comment:
I would rename the class to `CopyContentBehavior` as it doesn't copy the clipboard, but the content.